include/version.cpp

```diff
@@ -146,7 +146,7 @@ static bool write_version_h(const std::string &versionh, const std::map<std::str
 		return false;
 
 	fd << "#pragma once" << std::endl;
-	for (std::map<std::string, std::string>::const_iterator it = versions.begin(); it != versions.end(); ++it)
+	for (auto it = versions.begin(); it != versions.end(); ++it)
```

Comment:
If you're going to update things like this you may as well just make it use a range based for loop.

```suggestion
	for (const auto &[key, value] : versions)
```

Reply:
@SadieCat,
Ure absolutely right, I'll fix it.

Reply:
@SadieCat, fix it.
https://github.com/anope/anope/pull/295/commits/4fb0a3908ddd4305f06db76d8e60330a92e4a28e